### PR TITLE
Add Let’s Encrypt to SSL certificates tip

### PR DESCRIPTION
**Pull request by p-bo**
Signed-off-by: p-bo <pavel.borecki@gmail.com>

### CODE REVIEW

**Comment by kesselb** [comment]
Hi, thanks for your pull request :+1: 

May I suggest rephrasing the block like: 

> Self-signed certificates have their drawbacks - especially when you plan to make your Nextcloud server publicly accessible. Consider getting a certificate signed by a signing authority.

**Comment by p-bo** [comment]
Done :-)

File: admin_manual/installation/source_installation.rst [hunk 1]
```diff
@@ -263,11 +263,11 @@ the default site. Open a terminal and run::
...
 .. note:: Self-signed certificates have their drawbacks - especially when you
-          plan to make your Nextcloud server publicly accessible. You might
-          want to consider getting a certificate signed by a commercial signing
-          authority. Check with your domain name registrar or hosting service
-          for good deals on commercial certificates.
-
+          plan to make your Nextcloud server publicly accessible. Consider getting
+          a certificate signed by a signing authority. Check with your domain name
+          registrar or hosting service for good deals on commercial certificates.
+          Or use free `Let's Encrypt <https://letsencrypt.org/>`_ one.
```
Comment (tflidd) [inline admin_manual/installation/source_installation.rst:269]: last line:
Or use **a** free...
Or use free Let's encrypt one**s**.

Comment (p-bo) [reply admin_manual/installation/source_installation.rst]: Agree, sorry for that - corrected.

**Comment by tflidd** [comment]
If you remove it, the link to Let's encrypt is missing and people might start asking, where they can get certificates from. Or just refer to the free ones, companies that prefer commercial ones probably know where to get them?

**Review by tflidd — APPROVED** [review]

**Comment by kesselb** [comment]
> If you remove it, the link to Let's encrypt is missing and people might start asking, where they can get certificates from.

This argumentation works for everything. Where to get a server, What is linux, etc. I would prefer relevant information in our documentation for operating Nextcloud. The certificate is your business. Google or ChatGPT can help you.

**Pull request merged**